services/server/src/config/default.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     secret: process.env.SESSION_SECRET || "CHANGE_ME",
     maxAge: 12 * 60 * 60 * 1000, // 12 hrs in millis
     secure: false, // Set Secure in the Set-Cookie header i.e. require https
+    storeType: "memory",
```

Comment:
This should be explained to other config files too. 

Also whenever we change anything in configs it must be documented in docs

```suggestion
    storeType: "memory", // Where to save the session info. "memory" is only good for testing and local development. Don't use it in production!
```